### ISSUE TITLE
Tiled Gallery Gutenberg and Site Accelerator

### ISSUE THREAD

**Issue by GuillermoArg** [Automattic/jetpack#19658]
#### Is your feature request related to a problem? Please describe.

You no can revert the tiled gallery in Gutenberg to standard wordpress gallery, you need make it by hand, one by one
And you no can use Tiled Gallery without the CDN (also if you no enable the CDN images)
People who use the new tiled gallery no know that is different to the old tiled.
And I think, how I no enable the site accelerator images and script files, tiled gallery will use my server images,
So you never enabled the CDN but Tiled gallery use it, and if you want no use more the tiled gallery, you need replace it by hand, that was my problem.

#### Describe the solution you'd like

This is one easy solution
I want 2 options in wordpress media or jetpack desk:
One option is for transform the standard gallery to tiled gallery and choice (mosaic, circle, etc)
Other option is for use or no CDN images, use CDN or use the different standard size images, small medium large
Can be another extra option for transform the actual tiled gallery to standard wordpress gallery and no replace it by hand one by one.

#### Describe alternatives you've considered

One alternative necessary is one buton in jetpack desk for revert the tiled gallery in gutenberg to standard wordpress gallery, something we lost from the old tiled gallery (and was great because you no broke your wordpress galleries).

#### Additional context

And now the gutenberg is the standard in wordpress, no many users use the classic editor.
You are ingeneer but no think like one user...
The idea is make Wordpress easy, I am System Analyst and like ergonomics, I like to contribute with jetpak and wordpress, :)

Thank you and have a nice day!

**Comment by jeherve** [comment]
> You no can revert the tiled gallery in Gutenberg to standard wordpress gallery

You should be able to convert a Tiled Gallery Block into a Gallery block like so:

https://user-images.githubusercontent.com/426388/116216671-23ccaf00-a749-11eb-923d-446369b9c8e8.mov

You will need to do it for all existing galleries though. As you indeed mentioned, there is currently no setting to update the content of multiple posts at once.

> you no can use Tiled Gallery without the CDN (also if you no enable the CDN images)

That's correct. Tiled Galleries (whether you use the block editor or the old Tiled Galleries in the classic editor) use Jetpack's Image CDN so the images can be resized to create the tiled effect without impacting your site's performance. If Jetpack did not use the image CDN, it would have to load full-size images, thus making your posts load slower.

**Comment by GuillermoArg** [comment]
Yes you can revert one by one, but also I disabled the cdn in the modules and in some galleries no appear the block in Gutenberg..error we no can load this block, something how that, but in the web appear, so I search the images and create the standard gallery from zero. I don't no why.

Is better left the user decide.
And you can use different sizes images for differents screens sizes... small or medium, is just some little kb.  Medium is 350 px (if you no change it)

I think is one good idea change the name to Tiled  CDN. So user know what they are using.
Why no enabled for example the Cdn images in the config? I has disable it but tiled continue using it.
So when you press disable cdn images, will continue on, and alert this is for the tiled gallery.
Or one new button On, CDN for Tiled Galleries, and you no can turn off.

One old plugin make this but is older, "tiled gallery without jetpack"

At least make one function or plugin for revert all automatically.

Is more simple 2 or 3  buttons in Media. 😐
Now I no are using this feature,
If I have another idea I will write,  the feature is very good, just need options.
Thanks for answer.  👍

**Comment by jeherve** [comment]
> I think is one good idea change the name to Tiled CDN. So user know what they are using.
> Why no enabled for example the Cdn images in the config?

That is indeed something where we could improve wording to make things clearer. This is something we've discussed in the past in #7576 

> you can use different sizes images for differents screens sizes... small or medium, is just some little kb. Medium is 350 px (if you no change it)

I'm afraid that is not the best solution, since this would limit the layouts that can be produced by the gallery to the different image sizes available on the site.

> One old plugin make this but is older, "tiled gallery without jetpack"

Indeed. That plugin suffers from the performance issues I mentioned earlier, though. By loading full-size images in each gallery, you potentially slow down loading time on pages with large images and / or galleries with lots of images.

> At least make one function or plugin for revert all automatically.

We'll consider adding an option, and update this issue when we have some news!

**Comment by ckubitza** [comment]
You can disable CDN, even for tiled gallery, with this filter:

add_filter('jetpack_photon_development_mode', '__return_true');

**Comment by HammyHavoc** [comment]
> You can disable CDN, even for tiled gallery, with this filter:
> 
> `add_filter('jetpack_photon_development_mode', '__return_true');`

The image thumbnails are still attempted to be fetched via URLs like `https://i0.wp.com/www.mydomain.com/wp-content` even with this it seems.

**Comment by jeherve** [comment]
> The image thumbnails are still attempted to be fetched via URLs like https://i0.wp.com/www.mydomain.com/wp-content even with this it seems.

That's correct. As I mentioned above, Tiled Galleries always use Jetpack's Image CDN. That filter will not change that.
